### PR TITLE
Shodo-1.2.0 version 1.2.0

### PR DESCRIPTION
**Pull request by wendak478**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108217)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by Trenly** [comment]
[Policy] Dependencies

**Comment by borque** [comment]
The packages name looks a bit strange to me. Does ist comply with wingets naming schema, or is it that shodo is the name, - is the seperator and 1.2.0 is the version?

**Comment by stephengillie** [comment]
Internet searches suggest that this is a digital signing and/or legal application released by `PJe` (Processo Judicial Eletrônico) in the [Brasilian legal system](https://www.tst.jus.br/web/pje/acesse-o-pje). 

"Shodō is an application used to access the PJe and sign petitions and documents."

File: manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0
+PackageVersion: 1.2.0
+PackageLocale: en-US
+Publisher: Assinador Shodo
+PackageName: Shodo-1.2.0
```
Comment (stephengillie) [inline manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml:8]:
```suggestion
Publisher: Pje
PublisherUrl: https://ww2.trt2.jus.br/servicos/acesso-online/processo-judicial-eletronico-pje/page
PackageName: PJe.Sjodo
PackageUrl: https://www2.trt8.jus.br/PJe/Shodo%20-%20Processo%20Judicial%20Eletr%C3%B4nico.htm
```
"Assinador" just means "signer".

File: manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0
+PackageVersion: 1.2.0
+PackageLocale: en-US
```
Comment (stephengillie) [inline manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml:6]:
```suggestion
PackageLocale: pt-BR
```

File: manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0
+PackageVersion: 1.2.0
+DefaultLocale: en-US
```
Comment (stephengillie) [inline manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.yaml:6]:
```suggestion
DefaultLocale: pt-BR
```

File: manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0
```
Comment (stephengillie) [inline manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.yaml:4]:
```suggestion
PackageIdentifier: PJe.Sjodo
```

File: manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0
```
Comment (stephengillie) [inline manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.installer.yaml:4]:
```suggestion
PackageIdentifier: PJe.Sjodo
```
Please update file name and path to match this change.

File: manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml:8]:
```suggestion
PackageIdentifier: PJe.Sjodo
PackageVersion: 1.2.0
PackageLocale: pt-BR
Publisher: Pje
PublisherUrl: https://ww2.trt2.jus.br/servicos/acesso-online/processo-judicial-eletronico-pje/page
PackageName: PJe.Sjodo
PackageUrl: https://www2.trt8.jus.br/PJe/Shodo%20-%20Processo%20Judicial%20Eletr%C3%B4nico.htm
```
Please update file name and path to match this change. 
"Assinador" just means "signer".

**Comment by Trenly** [comment]
Dependencies are now supported in the 1.6 client. If you could please submit a new manifest for the latest version of the package, it would be greatly appreciated!

**Pull request closed without merging**